### PR TITLE
Feature/better docker

### PR DESCRIPTION
**Pull request by JeremyRms**
What you guys think?
Should we drop PHP from frontend container?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,3 +32,6 @@ unit:
 feature:
 	@echo "$(shell echo "\033[0;49;32m---\033[m")" Feature Test
 	cd backend && vendor/bin/phpunit --testsuite=Feature
+
+composer-install:
+	cd backend && composer install
```
Comment (nootanghimire) [inline Makefile]: Add newline on EOF please! :)

File: backend/composer.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-    "description": "The Laravel Framework.",
-    "keywords": ["framework", "laravel"],
+    "name": "opendesign/opendesign",
+    "description": "OpenDesign",
+    "keywords": ["amazing", "stuff"],
     "license": "MIT",
     "type": "project",
     "require": {
-        "php": ">=7.0.0",
+        "php": "7.1.*",
```
Comment (nootanghimire) [inline backend/composer.json]: maybe we need `^7.1` ? What do you think?

**Review by nootanghimire — APPROVED** [review]
Some minor changes,

**Comment by nootanghimire** [comment]
@JeremyZr  I would say we remove php from frontend. Maybe only go with nginx and react.

**Comment by JeremyRms** [comment]
Updated, please review

**Review by nootanghimire — APPROVED** [review]

**Comment by nootanghimire** [comment]
Merged at https://github.com/apacig/opendesign/commit/d5f1261a8d4230d32e39849a9ca4db257d3ad07a

**Pull request merged**